FinderKeeper/Assets/Scripts/Direction.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Direction : MonoBehaviour {
+    public float RotationSpeed;
+    private Quaternion _lookRotation;
+    private Vector3 _direction;
+
+    public GameObject FindClosestItem()
+    {
+        GameObject[] items;
+        items = GameObject.FindGameObjectsWithTag("Low");
+        GameObject closest = null;
+        float distance = Mathf.Infinity;
+        Vector3 position = transform.position;
+        foreach (GameObject item in items)
+        {
+            Vector3 diff = item.transform.position - position;
+            float curDistance = diff.sqrMagnitude;
+            if (curDistance < distance)
```

Comment:
Could you explain this line please? Won't it always return true the first time?

Reply:
I'm assuming you are referring to line 21. As distance = infinity, it checks whether the distance between the item and the pointer is => infinity (which it shouldn't be), so yeah, it should always return true.